code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -96,6 +96,7 @@
 	AddElement(/datum/element/simple_flying)
 	ADD_TRAIT(src, TRAIT_SPACEWALK, INNATE_TRAIT)
 	ADD_TRAIT(src, TRAIT_NO_FLOATING_ANIM, INNATE_TRAIT)
+	ADD_TRAIT(src, TRAIT_BOMBIMMUNE, INNATE_TRAIT)
```

Comment:
this won't do anything. TRAIT_BOMBIMMUNE is only checked for in the ex_act() code for HUMANS, and even there, it needs to be a part of your species's species_traits list in order to do anything.

Reply:
this probably means that TRAIT_BOMBIMMUNE does nothing for rust heretics either, unless I'm misreading the code.

Reply:
Well, have you ever tried bombing a rust heretic on rust tiles?